cassandra/cassandra-docker.sh

```diff
@@ -34,38 +34,29 @@ do
     --seed_provider_classname=*)
       SEED_PROVIDER_CLASSNAME="${args#*=}"
     ;;
-    --internode_encryption=*)
```

Comment:
I did some reorg on the options here. If that's not acceptable, possibly to keep some backwards compatibility, I can certainly try to retrofit.